spec/support/devise.rb

```diff
@@ -1,4 +1,5 @@
 RSpec.configure do |config|
     config.include Devise::Test::ControllerHelpers, type: :controller
     config.include Devise::Test::ControllerHelpers, type: :view
+    config.include Devise::Test::IntegrationHelpers, type: :request
```

Comment:
Out of curiosity because I don't know what this is for, why is this configuration needed?

Reply:
This was how Erik helped us solve a problem with testing the user authenticated controllers.  Because the tests we were running had the type request, we had to config to include that type.